gemma/layers.py

```diff
@@ -34,12 +34,14 @@ class RMSNorm(nn.Module):
 
   @nn.compact
   def __call__(self, x):
-    scale = self.param('scale', nn.initializers.zeros_init(), (x.shape[-1]))
+    scale = self.param('scale', nn.initializers.ones, (x.shape[-1], ))
```

Comment:
Explain this line, why do need this change ?

Reply:
The previous implementation initialized `scale` with zeros, which would nullify the effect of scaling. By changing the initializer to ones, the scale parameter will start with a neutral effect, ensuring that the layer's initial behavior is properly scaled inputs.

Reply:
thanks for clarifying this

Reply:
You're welcome!